metadata/index.html

```diff
@@ -1920,42 +1918,28 @@
       </ul>
       <p>
         A description object B is <dfn>merged</dfn> into an original description object A by merging each property of B into A. If the property from B does not exist on A, it is simply added to A. If A does have the property, the way the values are merged depends on the type of the property, as follows:
       </p>
       <ul>
         <li>If the property is an <a>array property</a>, the way in which values are merged depends on the property; see the relevant property for this definition.</li>
-        <li>If the property is a <a>link property</a>, <a>URI template property</a>, or <a>column reference property</a>, the value from A overrides that from B.</li>
-        <li>If the property is an <a>object property</a>, then if the property only accepts single objects:
-          <ol>
-            <li>if the value of the property in A is a string or the value from B is a string then the value from A overrides that from B</li>
-            <li>otherwise (if both values as objects) the objects are merged as described here</li>
-          </ol>
-          If the property accepts arrays, the result is an array of objects or strings: those from A followed by those from B that were not already a value in A.</li>
-        <li>If the property is a <a>natural language property</a>, the result is an object whose properties are language codes and where the values of those properties are arrays. The arrays MUST provide the values from A followed by those from B that were not already a value in A. If a value exists in <code>und</code> and any other language</code> the value from <code>und</code> MUST be removed, ensuring that the <code>und</code> property is removed if it's value becomes an empty array.</li>
-        <li>If the property is an <a>atomic property</a>, then if the property only accepts single values, the value from A overrides that from B; otherwise the result is an array of values: those from A followed by those from B that were not already a value in A.</li>
+        <li>If the property is an <a>object property</a> the objects are merged as described here.</li>
+        <li>If the property is a <a>natural language property</a>, the result is an object whose properties are language codes and where the values of those properties are arrays. The arrays MUST provide the values from A followed by those from B that were not already a value in A. If a value exists in <code>und</code> and any other language the value from <code>und</code> MUST be removed, ensuring that the <code>und</code> property is removed if it's value becomes an empty array.</li>
         <li>If the property is a <a>common property</a>, the result is an array containing values from A followed by values from B not already in A.</li>
```

Comment:
Do we need to say something here about how object equality is assessed, when a common property has an object as a value? Is it deep equality?